### PR TITLE
MULTIARCH-4784: Multi-version configuration for the multiarch-tuning-operator FBC

### PR DESCRIPTION
**Pull request by aleskandro**
This PR configure a multi-version project for the FBC of the multiarch-tuning-operator, starting from 4.17 and leaving 4.16 to the UI/manual creation

### CODE REVIEW

**Comment by ernesgonzalez33** [comment]
I'm not an expert here. @ifireball do you mind checking this PR?

File: auto-generated/cluster/stone-prd-rh01/tenants/multiarch-tuning-ope-tenant/projctl.konflux.dev_v1beta1_projectdevelopmentstreamtemplate_file-based-catalog.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          devfileUrl: https://raw.githubusercontent.com/openshift/multiarch-tuning-operator/fbc-4.16/devfile.yaml
+          revision: fbc-{{.version}}
+          url: https://github.com/openshift/multiarch-tuning-operator
+      targetPort: 50051
+  variables:
+  - description: A version number for a new OCP stream
+    name: version
+  - description: |
+      The name of some objects cannot include dots. It's recommended to set this versionName replacing any `.` with hyphen (`-`) in the value of the parameter `version`.
+    name: versionName
```
Comment (ifireball) [inline auto-generated/cluster/stone-prd-rh01/tenants/multiarch-tuning-ope-tenant/projctl.konflux.dev_v1beta1_projectdevelopmentstreamtemplate_file-based-catalog.yaml:59]: You might need an extra newline in the string above this to ensure this is not included in it

**Review by ifireball — APPROVED** [review]
Some minor yaml formatting issues like missing eof newlines. Otherwise, lgtm

File: auto-generated/cluster/stone-prd-rh01/tenants/multiarch-tuning-ope-tenant/projctl.konflux.dev_v1beta1_projectdevelopmentstreamtemplate_file-based-catalog.yaml (hunk 1 above)
Comment (aleskandro) [reply auto-generated/cluster/stone-prd-rh01/tenants/multiarch-tuning-ope-tenant/projctl.konflux.dev_v1beta1_projectdevelopmentstreamtemplate_file-based-catalog.yaml:59]: This should be a rendering issue in the GH UI.

What you read here is generated correctly from https://github.com/redhat-appstudio/tenants-config/pull/475/files#diff-03b792a4bde4e83776f271e135ded766199eac60d2001e0fb41cc1fe29213a83R11

**Pull request merged**